src/test/scripts/functions/transform/input/homes3/homes.tfspec_hash.json

```diff
@@ -0,0 +1,2 @@
+{
+    "ids": true, "recode": [ 1, 2, 7 ], "K": 100 }
```

Comment:
It seems there is a typo in this json specification ("recode" instead of "hash")